go.mod

```diff
@@ -6,135 +6,192 @@ toolchain go1.25.1
 
 require (
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
-	github.com/gardener/gardener v1.117.6
-	github.com/gardener/gardener-extension-provider-openstack v1.47.0
+	github.com/gardener/gardener v1.127.1
+	github.com/gardener/gardener-extension-provider-openstack v1.49.0
 	github.com/go-logr/logr v1.4.3
 	github.com/golang/mock v1.6.0
-	github.com/onsi/ginkgo/v2 v2.25.3
-	github.com/onsi/gomega v1.38.2
+	github.com/onsi/ginkgo/v2 v2.25.0
+	github.com/onsi/gomega v1.38.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.10.1
 	github.com/spf13/pflag v1.0.10
 	golang.org/x/tools v0.37.0
 	gopkg.in/yaml.v3 v3.0.1
-	istio.io/api v1.25.2
+	istio.io/api v1.25.5
 	istio.io/client-go v1.25.1
-	k8s.io/api v0.32.9
-	k8s.io/apiextensions-apiserver v0.32.9
-	k8s.io/apimachinery v0.32.9
-	k8s.io/client-go v0.32.9
-	k8s.io/code-generator v0.32.9
-	k8s.io/component-base v0.32.9
+	k8s.io/api v0.33.5
+	k8s.io/apiextensions-apiserver v0.33.5
+	k8s.io/apimachinery v0.33.5
+	k8s.io/client-go v0.33.5
+	k8s.io/code-generator v0.33.5
+	k8s.io/component-base v0.33.5
 	k8s.io/utils v0.0.0-20250820121507-0af2bda4dd1d
-	sigs.k8s.io/controller-runtime v0.20.4
-	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20231015215740-bf15e44028f9
+	sigs.k8s.io/controller-runtime v0.21.0
+	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20250517180713-32e5e9e948a5
```

Comment:
We could drop this dependency in favor of installing it from the release artifacts similar to g/g